modules/control-utility/src/test/java/org/apache/ignite/util/SystemViewCommandTest.java

```diff
@@ -346,6 +346,7 @@ public void testServices() {
         assertEquals(srvcCfg.getName(), sysView.get(1)); // name
         assertEquals(DummyService.class.getName(), sysView.get(2)); // serviceClass
         assertEquals(Integer.toString(srvcCfg.getMaxPerNodeCount()), sysView.get(6)); // maxPerNodeCount
+        assertEquals(F.first(ignite0.services().serviceDescriptors()).topologySnapshot().toString(), sysView.get(10));
```

Comment:
Suggestion:
`for(Ignite ig : G.allGrids()){
            List<List<String>> srvsView = systemView((IgniteEx)ig, SVCS_VIEW);

            assertEquals(1, srvsView.size());

            List<String> sysView = srvsView.get(0);

            assertEquals(srvcCfg.getName(), sysView.get(1)); // name
            assertEquals(DummyService.class.getName(), sysView.get(2)); // serviceClass
            assertEquals(Integer.toString(srvcCfg.getMaxPerNodeCount()), sysView.get(6)); // maxPerNodeCount
            assertEquals(F.first(ig.services().serviceDescriptors()).topologySnapshot().toString(), sysView.get(10));
        }`
Up2U

Reply:
Removed the topology check on a second node